src/daemon/pex/pex.go

```diff
@@ -215,6 +229,33 @@ func New(cfg Config, defaultConns []string) (*Pex, error) {
 	return pex, nil
 }
 
+func (px *Pex) download() error {
+	resp, err := http.Get(PeerDatabaseUrl)
```

Comment:
Make the download and read into a separate function at package level, that accepts a url and returns a string

Reply:
Later, this will be moved to util and have backoff options